packages/search/src/content/index.ts

```diff
@@ -322,30 +326,37 @@ function performEnterpriseContentSearch(
   );
 }
 
+/**
+ * Internal function to search the Hub API (v3)
+ * using the same arguments as searchContent()
+ *
+ * NOTE: this returns the Hub API's raw JSONAPI response
+ * and invalid parameters like isPortal will be ignored
+ *
+ * @param request - see searchContent()
+ * @returns Hub API's JSONAPI response
+ *
+ * @private
+ */
+export function _searchHubByContentSearchRequest(
```

Comment:
I'm definitely open to renaming this if someone has a better idea.

Reply:
Is `_searchHubContent` no good?

Reply:
Thanks for the suggestion @drspacemanphd. I thought about `searchHubContent()`, but it doesn't return content. @andrewctate suggested `searchHubDatasets()`, which is kind of inline w/ what I was originally thinking of calling this: `searchDatasets()`.

He also suggested `searchHubApi()`, which to me is more appropriate for the `searchHub()` fn I added in this PR. Since that fn is a thin wrapper over `hubApiRequest()` I'm thinking of renaming it to `hubApiSearch()`, that way we can reserve the `search` prefix for fns that that `IHubSearchRequestOptions`, which brings me back around to naming this `searchDatasets()`.


I think I'm going to go w/